ratelimit/ratelimit.go

```diff
@@ -422,3 +423,11 @@ func newRatelimit(s Settings, sw Swarmer, redisRing *ring) *Ratelimit {
 		impl:     impl,
 	}
 }
+
+func Headers(s *Settings, retryAfter int) http.Header {
+	limitPerHour := int64(s.MaxHits) * int64(time.Hour) / int64(s.TimeWindow)
```

Comment:
Calculates limit in `int64`. Before it was 
```go
settings.MaxHits * int(time.Hour/settings.TimeWindow)
```
which would produce incorrect result when `settings.TimeWindow > time.Hour`, e.g. `MaxHits = 2, TimeWindow = "2h"` would produce 0, whilte it should be 1.